packages/ketcher-react/src/script/editor/Editor.ts

```diff
@@ -577,17 +577,18 @@ function isMouseRight(event) {
 function domEventSetup(editor: Editor, clientArea) {
   // TODO: addEventListener('resize', ...);
   ;[
-    'click',
-    'dblclick',
-    'mousedown',
-    'mousemove',
-    'mouseup',
-    'mouseleave',
-    'mouseover'
-  ].forEach((eventName) => {
+    { target: clientArea, eventName: 'click' },
```

Comment:
Thanks for the idea! If it is to be extracted as a standalone variable, that definitely yes. Otherwise I don't see a benefit: it is not reused, it is not a prop, and the type is easily inferred.

What I would do is move all eventNames to a enum, but it should involve refactoring multiple locations and I think deserve a separate PR.